### PR TITLE
Add com.github.gmg137.netease-cloud-music-gtk

### PR DESCRIPTION
**Pull request by catsout**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://github.com/gmg137/netease-cloud-music-gtk/issues/133**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by catsout** [comment]
bot, build com.github.gmg137.netease-cloud-music-gtk

**Comment by catsout** [comment]
bot, build com.github.gmg137.netease-cloud-music-gtk

File: com.github.gmg137.netease-cloud-music-gtk.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-music
+  - --persist=.lyrics
+  - --metadata=X-DConf=migrate-path=/com/github/gmg137/netease-cloud-music-gtk/
```
Comment (barthalion) [inline com.github.gmg137.netease-cloud-music-gtk.yml:19]: Is it actually needed, or just keeping it here for the sake of pre-flatpak installs?

File: com.github.gmg137.netease-cloud-music-gtk.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+app-id: com.github.gmg137.netease-cloud-music-gtk
```
Comment (barthalion) [inline com.github.gmg137.netease-cloud-music-gtk.yml:1]: `io.github` is usually preferred, but not a blocker.

File: com.github.gmg137.netease-cloud-music-gtk.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline com.github.gmg137.netease-cloud-music-gtk.metainfo.xml:1]: Ideally should be upstreamed.

File: com.github.gmg137.netease-cloud-music-gtk.yml (hunk 2 above)
Comment (catsout) [reply com.github.gmg137.netease-cloud-music-gtk.yml:1]: I got this id from `dbus-monitor`

File: com.github.gmg137.netease-cloud-music-gtk.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: netease-cloud-music-gtk
+rename-icon: netease-cloud-music-gtk
+rename-desktop-file: netease-cloud-music-gtk.desktop
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
```
Comment (hfiguiere) [inline com.github.gmg137.netease-cloud-music-gtk.yml:14]: I don't see wayland permssions.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.gmg137.netease-cloud-music-gtk.yml (hunk 1 above)
Comment (catsout) [reply com.github.gmg137.netease-cloud-music-gtk.yml]: Removed

Comment (catsout) [reply com.github.gmg137.netease-cloud-music-gtk.yml]: Removed

File: com.github.gmg137.netease-cloud-music-gtk.yml (hunk 4 above)
Comment (catsout) [reply com.github.gmg137.netease-cloud-music-gtk.yml:14]: So the `fallback-x11` should always used with `wayland`?

Comment (hfiguiere) [reply com.github.gmg137.netease-cloud-music-gtk.yml:14]: the name say "fallback". If the app supports wayland, then `--socket=wayland` to display on wayland and `--socket=fallback-x11` if wayland is not available and the app support a fallback to x11. (I no know app that don't support x11).

If the app doesn't support wayland then only `--socket=x11`

Comment (catsout) [reply com.github.gmg137.netease-cloud-music-gtk.yml:15]: Added.

**Comment by nedrichards** [comment]
bot, build com.github.gmg137.netease-cloud-music-gtk

**Pull request closed without merging**